### PR TITLE
setup initial schema and seed for events

### PR DESCRIPTION
**Pull request by foodaka**

### CODE REVIEW

File: API/entities/event.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
+
+
+class Event {
```
Comment (foodaka) [inline API/entities/event.js:3]: wip

File: README.md [hunk 2]
```diff
@@ -6,7 +6,45 @@ This repo contains the Deticket web application.
...
+
 ```sh
 yarn
 yarn dev
 ```
+
+Migrate the database
+
+```sh
+sequelize db:migrate
```
Comment (marbar3778) [inline README.md:19]: does this tool need to be installed?

File: README.md [hunk 3]
```diff
@@ -6,7 +6,45 @@ This repo contains the Deticket web application.
 
 To run the application in dev mode follow these instructions:
 
+Create a postgres database on your local machine and name it `detticket_dev`
```
Comment (marbar3778) [inline README.md:9]: want to open an issue to add a cmd with make or something else to do this automatically.

File: Dockerfile [hunk 4]
```diff
@@ -0,0 +1,13 @@
+FROM node:10
```
Comment (marbar3778) [inline Dockerfile:1]: is there a reason for node:10?

File: server.js [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+
+app.prepare().then(() => {
+  const server = express()
+  const httpServer = http.Server(server);
+
+  server.use(morgan(/* format string here */));
+
+  const db = require('./API/models');
+  const routes = require('./API/routes');
+  server.use(compression());
```
Comment (marbar3778) [inline server.js:29]: nice

File: package.json [hunk 6]
```diff
@@ -3,21 +3,42 @@
   "version": "0.1.0",
   "private": true,
   "scripts": {
-    "dev": "next dev -p 8080",
+    "dev": "node server.js",
```
Comment (marbar3778) [inline package.json:6]: this should include frontend as well

Comment (foodaka) [reply package.json:6]: this already does include frontend

Comment (foodaka) [reply package.json:6]: cause of nextjs is ssr

File: README.md (hunk 2 above)
Comment (foodaka) [reply README.md:19]: should just use docker

Comment (marbar3778) [reply README.md:19]: should state that in the readme

**Comment by marbar3778** [comment]
docker works someminor cleanup of this pr is needed but will do so in a following pr

File: Dockerfile [hunk 7]
```diff
@@ -0,0 +1,13 @@
...
+
+COPY ./package.json .
+
+RUN yarn 
+
+COPY . .
+
+EXPOSE 3000
+
+CMD yarn dev
```
Comment (foodaka) [inline Dockerfile:13]: BTW it seems the port 5432:5432 is fine. When changed to `RUN yarn dev` it could never connect with DB. Not sure why

Comment (foodaka) [reply Dockerfile:13]: @marbar3778

Comment (foodaka) [reply Dockerfile:13]: I think we need to setup an entrypoint.sh file that runs the DB seed + migration

Comment (foodaka) [reply Dockerfile:13]: in the container

Comment (marbar3778) [reply Dockerfile:13]: i wonder if we can just remove this line and it will still run because you are setting this in the docker-compose file

**Pull request merged**